meta-lmp-bsp/recipes-bsp/trusted-firmware-a/tf-a-fio-st.inc

```diff
@@ -22,3 +22,42 @@ TFA_PLATFORM  = "stm32mp1"
 TF_A_SUFFIX   = "stm32"
 TF_A_METADATA_JSON ?= "plat/st/stm32mp1/default_metadata.json"
 TF_A_SOC_NAME  = "${STM32MP_SOC_NAME}"
+
+do_deploy:append:stm32mp15-eval-sec() {
+    if [ -z "${STM32_CUBE_PATH}" ] || [ -z "${STM32_ROT_KEY_PATH}" ] || [ -z "${STM32_ROT_KEY_PASSWORD}" ]; then
+        bbwarn "TF-A BL2 binaries won't be signed, as STM32_CUBE_PATH/STM32_ROT_KEY_PATH/STM32_ROT_KEY_PASSWORD variables are not set correctly"
```

Comment:
We can probably use bbfatal here instead (and drop exit).

Reply:
I think we should permit to build unsigned binaries, but show an explicit message that they are no signed.

Reply:
Sorry, was under the impression that the exit in the following line was aborting the execution in a way, not returning, which is why the fatal suggestion.

So create a function and call the function inside this if, as then there is no need to exit or return.

Reply:
I've additionally introduced new variable `STM32_ROT_SIGN_ENABLE`, just to avoid this confusion.